js/ui/messageTray.js

```diff
@@ -824,15 +824,16 @@ MessageTray.prototype = {
         this._sources = [];
         Main.layoutManager.addChrome(this._notificationBin);
 
-		// Settings
+        // Settings
         this.settings = new Gio.Settings({ schema_id: "org.cinnamon.desktop.notifications" })
-		function setting(self, source, camelCase, dashed) {
-			function updater() { self[camelCase] = source.get_boolean(dashed); }
-			source.connect('changed::'+dashed, updater);
-			updater();
-		}
-		setting(this, this.settings, "_notificationsEnabled", "display-notifications");
+        function setting(self, source, camelCase, dashed) {
```

Comment:
Use simple code here. No need for a generic update mechanism and abstract introspection. Do it like it's done for bottom-notifications already.

Reply:
I have nothing to do with this part of the code. I'm just fixing the indentation there from tabs to spaces.